### DISCUSSION TITLE
A question about WW3 output

### DISCUSSION THREAD

**Discussion by Agoni00** [NOAA-EMC/WW3#1065] [Q&A]
I set the output frequency of the WW3 to 24 hours. Is the output variable instantaneous or the average of 24 hours? Looking forward to your reply.

**Comment by mickaelaccensi** [comment]
Hi Agoni00,
it is instantaneously field
Best,

**Reply by Agoni00** [reply]
Hi mickaelaccensi
Thank you very much for your answer. What should I do if I want to get a 24-hour average field?
Best,

**Reply by mickaelaccensi** [reply]
I don't think it can be done as a directly output of WW3, therefore you need to produce for example hourly outputs and then average them to obtain a daily average.

**Reply by Agoni00** [reply]
Thank you for your reply.

**Reply by Agoni00** [reply]
Sorry, I have another question, when I set the WW3 output frequency to 24h in earth system mode, but it appeared two times in one day, namely 00000 and 01800. How is this going? As shown in the following figure:
![image](https://github.com/NOAA-EMC/WW3/assets/92639723/4f94600f-ec3a-4242-90e5-cfeb79648a22)

**Reply by mickaelaccensi** [reply]
you should double check your input files parameters because it should not produce this file at 01800

**Reply by Agoni00** [reply]
My wav_in file is setup like this:
![image](https://github.com/NOAA-EMC/WW3/assets/92639723/9aa9ee44-aa47-4027-9a93-304174e8738e)

**Reply by mickaelaccensi** [reply]
Unfortunately I don't use the Earth System Model so I don't know the parameters from this namelist. You should ask help from the IT of CESM
